catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/ListApplication.java

```diff
@@ -108,7 +110,8 @@ public void init() {
 
           List<Part> parts = new ArrayList<>(request.raw().getParts());
 
-          AttachmentInfo attachmentInfo = parseAttachments(parts);
+          Pair<AttachmentInfo, Metacard> attachmentInfo =
```

Comment:
Is the metacard in the pair used? Seems like you are only calling `getLeft`. In that case, don't save the pair, just the AttachmentInfo.

Reply:
Yes, in `attemptToSplitAndStore()`.